File: app/src/main/java/io/tanjundang/study/knowledge/selector/SelectorActivity.java
package io.tanjundang.study.knowledge.selector;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.RadioButton;
import android.widget.RadioGroup;

import io.tanjundang.study.R;
import io.tanjundang.study.base.BaseActivity;

public class SelectorActivity extends BaseActivity implements RadioGroup.OnCheckedChangeListener {

    RadioButton rbOne;
    RadioButton rbTwo;
    RadioGroup radioGroup;


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }


    @Override
    public void initView() {
        setContentView(R.layout.activity_selector);
        rbOne = (RadioButton) findViewById(R.id.rbOne);
        rbTwo = (RadioButton) findViewById(R.id.rbTwo);
        radioGroup = (RadioGroup) findViewById(R.id.radioGroup);
        radioGroup.setOnCheckedChangeListener(this);
    }

    @Override
    public void initData() {

    }

    @Override
    public void onCheckedChanged(RadioGroup group, int checkedId) {
//        if (checkedId == rbOne.getId()) {
//            Functions.toast(rbOne.getText().toString());
//        } else if (checkedId == rbTwo.getId()) {
//            Functions.toast(rbTwo.getText().toString());
//        }
    }
}

File: app/src/main/java/io/tanjundang/study/knowledge/selector/SelectorActivity.kt
package io.tanjundang.study.knowledge.selector

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.RadioButton
import android.widget.RadioGroup

import io.tanjundang.study.R
import io.tanjundang.study.base.BaseActivity

class SelectorActivity : BaseActivity(), RadioGroup.OnCheckedChangeListener {

    internal var rbOne: RadioButton? = null
    internal var rbTwo: RadioButton? = null
    internal var radioGroup: RadioGroup? = null


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
    }


    public override fun initView() {
        setContentView(R.layout.activity_selector)
        rbOne = findViewById<RadioButton>(R.id.rbOne)
        rbTwo = findViewById<RadioButton>(R.id.rbTwo)
        radioGroup = findViewById<RadioGroup>(R.id.radioGroup)
        radioGroup!!.setOnCheckedChangeListener(this)
    }

    public override fun initData() {

    }

    override fun onCheckedChanged(group: RadioGroup, checkedId: Int) {
        //        if (checkedId == rbOne.getId()) {
        //            Functions.toast(rbOne.getText().toString());
        //        } else if (checkedId == rbTwo.getId()) {
        //            Functions.toast(rbTwo.getText().toString());
        //        }
    }
}
